### PR TITLE
Cosmosdb

### PR DESCRIPTION
**Pull request by lisawong19**

### CODE REVIEW

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-mapping/reference.yml](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/cosmosdb/docs-ref-mapping/reference.yml) | :white_check_mark:Succeeded | |
[docs-ref-services/cosmosdb.md](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/cosmosdb/docs-ref-services/cosmosdb.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C26%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201706262143138151-27%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-mapping/reference.yml](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/cosmosdb/docs-ref-mapping/reference.yml) | :white_check_mark:Succeeded | |
[docs-ref-services/cosmosdb.md](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/cosmosdb/docs-ref-services/cosmosdb.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C26%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201706262154136249-27%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/cosmosdb.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ms.service: cosmosdb
+---
+
+# Azure CosmosDB libraries for Python
+
+## Overview
+
+Use CosmosDB in your Python applications to store and query JSON documents in a NoSQL data store.
+
+Learn more about [Azure CosmosDB](https://docs.microsoft.com/azure/cosmos-db/introduction)
```
Comment (sptramer) [inline docs-ref-services/cosmosdb.md]: End sentence in `.`

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-mapping/reference.yml](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/cosmosdb/docs-ref-mapping/reference.yml) | :white_check_mark:Succeeded | |
[docs-ref-services/cosmosdb.md](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/cosmosdb/docs-ref-services/cosmosdb.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C26%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201706262205225916-27%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-mapping/reference.yml](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/cosmosdb/docs-ref-mapping/reference.yml) | :white_check_mark:Succeeded | |
[docs-ref-services/cosmosdb.md](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/cosmosdb/docs-ref-services/cosmosdb.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C27%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201706271918034357-27%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by lmazuel** [comment]
@sankethka @lisawong19 azure-mgmt-cosmosdb is out 
https://pypi.python.org/pypi/azure-mgmt-cosmosdb

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-mapping/reference.yml](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/cosmosdb/docs-ref-mapping/reference.yml) | :white_check_mark:Succeeded | |
[docs-ref-services/cosmosdb.md](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/cosmosdb/docs-ref-services/cosmosdb.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C28%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201706280002266439-27%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**